service/grpc/codelingo/codelingo.pb.go

```diff
@@ -1336,6 +1000,32 @@ func (x *codeLingoReviewServer) Recv() (*ReviewRequest, error) {
 	return m, nil
 }
 
+func _CodeLingo_Codemod_Handler(srv interface{}, stream grpc.ServerStream) error {
```

Comment:
don't use underscores in Go names; func _CodeLingo_Codemod_Handler should be _CodeLingoCodemodHandler